diff --git a/src/components/MaterialCommunityIcon.tsx b/src/components/MaterialCommunityIcon.tsx
index 52719bdbe10ea47ab029cdde3df440183795c7f6..bf7500aaa6917cfa5286c65c9aad95f54e05ee93 100644
--- a/src/components/MaterialCommunityIcon.tsx
+++ b/src/components/MaterialCommunityIcon.tsx
@@ -39,18 +39,9 @@ export const accessibilityProps: AccessibilityProps =
  */
 const loadIconModule = () => {
   try {
-    return require('@react-native-vector-icons/material-design-icons').default;
+    return require('react-native-vector-icons/MaterialCommunityIcons').default;
   } catch (e) {
-    try {
-      return require('@expo/vector-icons/MaterialCommunityIcons').default;
-    } catch (e) {
-      try {
-        return require('react-native-vector-icons/MaterialCommunityIcons')
-          .default;
-      } catch (e) {
-        return null;
-      }
-    }
+    return null;
   }
 };
 
